README.md

```diff
@@ -69,13 +69,13 @@ is generated by the script but can be overridden.  The following is a basic
 layout:
 
     # What is XYZ?
-    
+
```

Comment:
This is intentionally left indented to show that this whole indented block is one section of contiguous "code" (to help ensure that Markdown translates it properly into a single contiguous `<code>` element).